### PR TITLE
Fix currentExcitement typo

### LINKED ISSUE #26: Fix currentExcitment typo
### Description
- We currently have a typo on our Application collection. One of our records is `currentExcitment ` when it should be `currentExcitement ` (its missing an e).

### Acceptance Criteria
- Make sure new records get stored with proper spelling
- Make sure existing records have the proper spelling

### PR DESCRIPTION
**Pull request by beebzz**
Fixes #26 and #30, plus a bug with the reflection page where the excitement wasn't being recorded properly

### CODE REVIEW

File: graphql/api.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ export const createAccount = async (email, name, company, role, date, type, note
               company: $company  
               role: $role
               initialExcitement: 1
-              currentExcitment: 3
```
Comment (zeeberry) [inline graphql/api.js:80]: could we update this to pull in the actual excitement data instead of this hard coded number?

**Review by zeeberry — COMMENTED** [review]
I left a small question. let me know if you'd rather it be a separate issue. Also, just for my understanding did you have to go in and manually update the existing records? or was the schema update enough?

File: graphql/api.js (hunk 1 above)
Comment (beebzz) [reply graphql/api.js:80]: we could, it's sort of unrelated to this but should only take a few lines

Comment (beebzz) [reply graphql/api.js:80]: and yeah i manually updated the records considering that there's only a few

File: components/intake/excitement.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ const Excitement = ({onClick}) => {
   const company = 'Florish';
   const context = useContext(Context);
   const handleClick = (excitement) => {
+    excitement = options.findIndex(element => element.toLowerCase() === excitement) + 1;
```
Comment (zeeberry) [inline components/intake/excitement.jsx:10]: given we're using this functionality in 2 files, can we transform it into a reusable function?

**Review by zeeberry — COMMENTED** [review]
I added a small suggestion. Code looks good! 💪🏼 

However, I'm getting a consistent 500 error when creating a new user. Are you able to create a user successfully?

**Review by zeeberry — APPROVED** [review]
Odd. Not sure why i was seeing that 500 error earlier, but not seeing it anymore. 

:shipit:

**Pull request merged**